edge-util/test/Microsoft.Azure.Devices.Edge.Storage.Test/DbStoreBackupRestoreTest.cs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Storage.Test
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
+    using Moq;
+    using Xunit;
+
+    [Unit]
+    public class DbStoreBackupRestoreTest
+    {
+        readonly string backupFolder;
+
+        public DbStoreBackupRestoreTest()
+        {
+            string tempFolder = Path.GetTempPath();
+            this.backupFolder = Path.Combine(tempFolder, $"dbStoreTestBackup{Guid.NewGuid()}");
+            if (Directory.Exists(this.backupFolder))
+            {
+                Directory.Delete(this.backupFolder);
+            }
+
+            Directory.CreateDirectory(this.backupFolder);
+        }
+
+        ~DbStoreBackupRestoreTest()
+        {
+            if (!string.IsNullOrWhiteSpace(this.backupFolder) && Directory.Exists(this.backupFolder))
+            {
+                Directory.Delete(this.backupFolder, true);
+            }
+        }
+
```

Comment:
is creating a backup folder here a forgotten copy/paste code? I don't see it being used below. Also, I don't see the tests in this source super-useful, it doesn't check higher level behavior, but very low-level implementation specific things like call forwarding and parameter checking.

Reply:
Ah yes, looks like I missed removing the code blocks in the constructor and dispose.

Yes, I added those tests to provide code coverage. The code in the target source class itself is minimal and I wanted the unit test to only be localized to testing what only the source class itself is doing.

Tests for other classes cover high-level behavior.

Reply:
Removed this test class and its target source completely